Lib/Logic_layer/LL.c

```diff
@@ -10,66 +10,73 @@
 /**
   * @brief  converts text into color.
   * @param  text color with text see colortext array for reference.
-  * @retval returns a 8-bit color code.
+  * @retval returns a 8-bit color code or 0x01 for invalid/syntax color.
   */
 uint8_t LL_textToColor(char *text)
 {
-	uint8_t color;
+	uint8_t color = 0;
+	uint8_t black = 0;
 	int i = 0;
-	char colortext[AMOUNTOFCOLORS][20] = {{"zwart"}, {"blauw"},{"lichtblauw"},
-										 {"groen"}, {"lichtgroen"}, {"cyaan"},
-										 {"lichtcyaan"}, {"rood"}, {"lichtrood"},
-										 {"magenta"}, {"lichtmagenta"}, {"bruin"},
-										 {"geel"}, {"grijs"}, {"wit"}, {"roze"}};
+	char colortext[AMOUNTOFCOLORS][20] = {{"zwart"},{"lichtcyaan"},{"lichtmagenta"},
```

Comment:
20 is ook een beetje een vaag getal die niet duidelijk is, mischien ook even een define van maken

Reply:
is gewoon ruimte in de buffer voor de text